components/installation-telemetry/BUILD.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License-AGPL.txt in the project root for license information.
+
+packages:
+  - name: lib
```

Comment:
Not really sure how the build works here in truth. I put it in because it seemed to be necessary to get the `components/common-go` library in - when it wasn't in, the build failed.

If I've done something wrong with the configuration, please advise on how I can fix it

Reply:
Sounds odd. I tried it here https://werft.gitpod-dev.com/job/gitpod-build-clu-test.28 and it works even without `lib`. Was maybe another issue you had?

Reply:
That werft job isn't building the `installation-telemetry` component so I think I'm right in saying that it wouldn't error. I'll try on a separate branch without the `lib` and see what happens

Reply:
Ouch. My bad. :disappointed:

Reply:
I'm trying it in https://werft.gitpod-dev.com/job/gitpod-build-sje-telemetry-client-2.0

Reply:
You're right. No idea what was going on when I did it.

Will remove the `lib` section